content/docs/user-guide/external-dependencies.md

```diff
@@ -1,23 +1,27 @@
 # External Dependencies
 
-There are cases when data is so large, or its processing is organized in a way
-such that you would like to avoid moving it out of its external/remote location.
-For example from a network attached storage (NAS), processing data on HDFS,
-running [Dask](https://dask.org/) via SSH, or having a script that streams data
+There are cases when data is so large, or its processing is organized in such a
+way, that its preferable to avoid moving it from its original location. For
+example data on a network attached storage (NAS), processing data on HDFS,
+running [Dask](https://dask.org/) via SSH, or for a script that streams data
 from S3 to process it.
 
-External <abbr>dependencies</abbr> and
+External dependencies and
 [external outputs](/doc/user-guide/managing-external-data) provide ways to track
```

Comment:
track and version?

Reply:
Do we version dependencies? Only when they're outputs of a previous stage, I think?

Reply:
I guess we do keep track of their versions in any case, but don't really control those versions if they are external.

Reply:
This sentence is about `External dependencies and [external outputs]`

Reply:
True. Updating in 60e8055d.